src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js

```diff
@@ -199,10 +199,14 @@ const SuggestionFacet = compose(
                       onKeyDown={selectableList.handleKeyEvent}
                       placeholder={placeholder}
                       value={inputValue}
-                      aria-activedescendant={_.get(
-                        selectableList,
-                        'focusedItem.id',
-                      )}
+                      aria-activedescendant={
+                        active
```

Comment:
Unfortunately false gets turned into a string so it won't work. Has to be either null or undefined.